site/pages/dashboard.page.tsx

```diff
@@ -254,6 +254,11 @@ const Dashboard = ({
             <Link className="govuk-link" href="/view-all-disruptions?draft=true">
                 <h2 className="govuk-heading-s text-govBlue">Draft disruptions</h2>
             </Link>
+            {STAGE !== "prod" && STAGE !== "preprod" && (
```

Comment:
We should inject this via the getServerSideProps, the STAGE env var won't be available on the front end directly without adding it to the next config

Reply:
done